### PR TITLE
docs: add usage as library

### LINKED ISSUE #43: Document usage as library
This package should be easy to use as a library.

We should document all the steps needed to use it in a standard and easy way.

### PR DESCRIPTION
**Pull request by fzipi**
Signed-off-by: Felipe Zipitria <felipe.zipitria@owasp.org>

Fixes #43.

### CODE REVIEW

**Comment by anuraaga** [comment]
FWIW I would address the global config issues before documenting usage as a library. It's doable but not a good experience right now

**Comment by fzipi** [comment]
Have a branch for that, need to push it.

File: README.md [hunk 1]
```diff
@@ -368,5 +368,52 @@ The rule looks for an HTTP header named `X-CRS-Test` and writes its value to the
 
 You can configure the name of the HTTP header by setting the `logmarkerheadername` option in the configuration to a custom value (the value is case insensitive).
 
+## Library usage
+
+`go-ftw` can be used as a library also. Just include it in your project:
+```sh
+go get github.com/coreruleset/go-ftw
+```
+```go
```
Comment (theseion) [inline README.md:377]: I'd include the imports here as well, to make it clear which calls refer to go-ftw.

**Review by theseion — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (fzipi) [reply README.md:399]: Added imports.

**Review by theseion — APPROVED** [review]

**Pull request merged**